tests/integration/test_packages.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from os import path
+import pytest
+
+import utils
+
+
+@pytest.mark.parametrize("env_name", ["without_deps", "with_deps"])
```

Comment:
`with_deps` should not be called for gomod at the moment. We should either have a single parametrize decorator with all valid combinations or add a code path to skip if gomod + with_deps (to be removed when implemented).

Reply:
Alternatively, you could add the commits from  #296 to this PR, change the PR title and close #296

Reply:
I moved the commit for ` gomod with_deps` here.